### PR TITLE
add Rheinland Pfalz

### PR DESCRIPTION
**Pull request by MartinGer**
(colleague of cyroxx) 
Added crawler for Rheinland Pfalz school data, no ids yet.

### CODE REVIEW

File: mv.xls [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```
Comment (cyroxx) [inline mv.xls]: Please remove this file from your commit.

File: run.py [hunk 2]
```diff
@@ -206,20 +206,21 @@ def get_nrw():
...
-    yield runner.crawl(SchleswigHolsteinSpider)
-    yield runner.crawl(BerlinSpider)
+    # yield runner.crawl(BrandenburgSpider)
+    # yield runner.crawl(Bayern2Spider)
+    # yield runner.crawl(NiedersachsenSpider)
+    # yield runner.crawl(SachsenSpider)
+    # yield runner.crawl(SachsenAnhaltSpider)
+    # yield runner.crawl(ThueringenSpider)
+    # yield runner.crawl(SchleswigHolsteinSpider)
+    # yield runner.crawl(BerlinSpider)
```
Comment (cyroxx) [inline run.py]: Please leave those lines unchanged in your PR as it makes merging more complicated than it needs to be.

File: run.py [hunk 3]
```diff
@@ -206,20 +206,21 @@ def get_nrw():
...
 
 if __name__ == '__main__':
     crawl()
     reactor.run() # the script will block here until the last crawl call is finished
-    get_mv()
-    get_hamburg()
-    get_nrw()
+    # get_mv()
+    # get_hamburg()
+    # get_nrw()
```
Comment (cyroxx) [inline run.py]: Please leave those lines unchanged in your PR as it makes merging more complicated than it needs to be.

File: jedeschule/pipelines/school_pipeline.py [hunk 4]
```diff
@@ -37,21 +37,24 @@ def process_item(self, item, spider):
                             website=item.get('website'),
                             fax=item.get('fax'),
                             email=email,
-                            address=address)
+                            address=address
+                            )
```
Comment (cyroxx) [inline jedeschule/pipelines/school_pipeline.py]: Those bracket changes are not required for the PR, but im kind of okay with it.

Comment (k-nut) [reply jedeschule/pipelines/school_pipeline.py]: In order to keep this PR small, I'd actually prefer those brackets not to change.

File: jedeschule/pipelines/school_pipeline.py [hunk 5]
```diff
@@ -104,17 +111,26 @@ def process_item(self, item, spider):
                             phone=item.get('Telefon'),
                             )
         elif spider.name == 'brandenburg':
-            school = School(
```
Comment (k-nut) [inline jedeschule/pipelines/school_pipeline.py]: Please don't change the indentation for the `Brandenburg` section here. Adding Rheinland-Pfalz should be enough

**Pull request merged**